lms/djangoapps/grades/models.py

```diff
@@ -761,6 +764,7 @@ class Meta(object):
     user = models.ForeignKey(User, blank=True, null=True)
     comments = models.CharField(max_length=300, blank=True, null=True)
     created = models.DateTimeField(auto_now_add=True, db_index=True)
+    earned_graded_override = models.FloatField(null=True, blank=True)
```

Comment:
is this to add support for grades overrides? is not it supported in Ironwood out of the box? @luismorenolopera

Reply:
That value is only called in a log, but it was not stored.